classes/controller/FrontController.php

```diff
@@ -316,7 +316,12 @@ public function init()
 
         /* Theme is missing */
         if (!is_dir(_PS_THEME_DIR_)) {
-            throw new PrestaShopException((sprintf(Tools::displayError('Current theme unavailable "%s". Please check your theme directory name and permissions.'), basename(rtrim(_PS_THEME_DIR_, '/\\')))));
+            throw new PrestaShopException(
+                $this->trans(
+                    'Current theme unavailable "%s". Please check your theme directory name and permissions.',
```

Comment:
Suggestion: `Current theme is unavailable. Please check your theme's directory name ("%s") and permissions.`